tests/test_plots.py

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
@steven-murray please look into this. It is definitely not complete.

Reply:
I think you can get away with just mocking the models that go into the plotting function. Just make a dictionary with a few "keys" in it, which hold dictionaries full of data (something like this:)

```python
models = {
    key: {
        'model': np.random.normal(size=(20, 50))
    } for key in ['one', 'two']
}
```